<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellmlAnnotationViewMetadataListWidget</class>
 <widget class="QWidget" name="CellmlAnnotationViewMetadataListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </layout>
  <action name="actionAddMetadata">
   <property name="icon">
    <iconset resource="../res/CellMLAnnotationView.qrc">
     <normaloff>:/oxygen/actions/list-add.png</normaloff>:/oxygen/actions/list-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Metadata</string>
   </property>
   <property name="statusTip">
    <string>Add a metadata</string>
   </property>
  </action>
  <action name="actionRemoveMetadata">
   <property name="icon">
    <iconset resource="../res/CellMLAnnotationView.qrc">
     <normaloff>:/oxygen/actions/list-remove.png</normaloff>:/oxygen/actions/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove Metadata</string>
   </property>
   <property name="statusTip">
    <string>Remove the current metadata or all the metadata</string>
   </property>
  </action>
  <action name="actionRemoveCurrentMetadata">
   <property name="text">
    <string>&amp;Current</string>
   </property>
   <property name="statusTip">
    <string>Remove the current metadata</string>
   </property>
  </action>
  <action name="actionRemoveAllMetadata">
   <property name="text">
    <string>&amp;All</string>
   </property>
   <property name="statusTip">
    <string>Remove all the metadata</string>
   </property>
  </action>
  <action name="actionClearMetadata">
   <property name="icon">
    <iconset resource="../../../../../res/ui.qrc">
     <normaloff>:/oxygen/actions/edit-delete.png</normaloff>:/oxygen/actions/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear Metadata</string>
   </property>
   <property name="statusTip">
    <string>Clear the current metadata or all the metadata</string>
   </property>
  </action>
  <action name="actionClearCurrentMetadata">
   <property name="text">
    <string>&amp;Current</string>
   </property>
   <property name="statusTip">
    <string>Clear the current metadata</string>
   </property>
  </action>
  <action name="actionClearAllMetadata">
   <property name="text">
    <string>&amp;All</string>
   </property>
   <property name="statusTip">
    <string>Clear all the metadata</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../../res/ui.qrc"/>
  <include location="../res/CellMLAnnotationView.qrc"/>
 </resources>
 <connections/>
</ui>
